tests/kernel/iop/vpool/pallocate.c

```diff
@@ -0,0 +1,86 @@
+#include <common-iop.h>
+#include <thpool.h>
+#include "vpl-common.h"
+
+#define MAX_ALLOC_SIZES 24
```

Comment:
Hmm, I would just use an `ARRAY_SIZE` (e.g. `sizeof(x) / sizeof(x[0])`) macro, and make the array dynamically sized.  Doesn't matter much, though...

-[Unknown]